### DISCUSSION TITLE
Handlebars from external URL not working in Magento

### DISCUSSION THREAD

**Discussion by katnavipasha** [handlebars-lang/handlebars.js#1883] [Q&A]
Hi, I am working on a module for an external app in Magento. Loading some external JS files from the external APP to add some HTML to the page. And Handlebars JS library and usage both are in that external JS file. But in magento it says "Uncaught ReferenceError: Handlebars is not defined". Can anyone pls help. Its been months I am trying to fix but nothing found yet.

**Comment by jaylinski** [comment]
See https://github.com/handlebars-lang/handlebars.js/discussions/1881.
